### PR TITLE
Create cloud-ops-win.yaml

### PR DESCRIPTION
**Pull request by smaksimov**

### CODE REVIEW

File: examples/OSPolicyAssignments/console/cloud-ops-win.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  resources:
+    - id: exec-script
+      exec:
+        validate:
+          script: |-
+            (New-Object Net.WebClient).DownloadFile("https://dl.google.com/cloudagents/add-google-cloud-ops-agent-repo.ps1", "${env:UserProfile}\add-google-cloud-ops-agent-repo.ps1")
+            Invoke-Expression "${env:UserProfile}\add-google-cloud-ops-agent-repo.ps1 -AlsoInstall -Version 2.*.*"
+            if ($LastExitCode) {
+            exit 101
+            }else {
```
Comment (manjunathganga) [inline examples/OSPolicyAssignments/console/cloud-ops-win.yaml:13]: nit:  add space after `{`

File: examples/OSPolicyAssignments/console/cloud-ops-win.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+            exit 100
+            }
+          interpreter: POWERSHELL
+        enforce:
+          script: |-
+            (New-Object Net.WebClient).DownloadFile("https://dl.google.com/cloudagents/add-google-cloud-ops-agent-repo.ps1", "${env:UserProfile}\add-google-cloud-ops-agent-repo.ps1")
+            Invoke-Expression "${env:UserProfile}\add-google-cloud-ops-agent-repo.ps1 -AlsoInstall -Version 2.*.*"
+            if ($LastExitCode) {
+            exit 101
+            }else {
```
Comment (manjunathganga) [inline examples/OSPolicyAssignments/console/cloud-ops-win.yaml:23]: nit:  add space after `{`

File: examples/OSPolicyAssignments/console/cloud-ops-win.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+id: google-cloud-ops-agent-windows-policy
+mode: ENFORCEMENT
+resourceGroups:
+  resources:
+    - id: exec-script
+      exec:
+        validate:
+          script: |-
+            (New-Object Net.WebClient).DownloadFile("https://dl.google.com/cloudagents/add-google-cloud-ops-agent-repo.ps1", "${env:UserProfile}\add-google-cloud-ops-agent-repo.ps1")
+            Invoke-Expression "${env:UserProfile}\add-google-cloud-ops-agent-repo.ps1 -AlsoInstall -Version 2.*.*"
```
Comment (adjackura) [inline examples/OSPolicyAssignments/console/cloud-ops-win.yaml:10]: This can't be right, there should be some other flag here to specify validate phase

**Review by manjunathganga — COMMENTED** [review]
Can you also add this example to the gcloud section: https://github.com/GoogleCloudPlatform/osconfig/tree/master/examples/OSPolicyAssignments/gcloud

You can use the cloudops centos example as a reference:
https://github.com/GoogleCloudPlatform/osconfig/blob/master/examples/OSPolicyAssignments/gcloud/cloudops-centos.yaml

Remember to replace `centos` with `windows` here and test with gcloud https://github.com/GoogleCloudPlatform/osconfig/blob/master/examples/OSPolicyAssignments/gcloud/cloudops-centos.yaml#L45

**Review by adjackura — CHANGES_REQUESTED** [review]
Seperate the script downloading into a file resource, and make sure to use dry run with exec resource validation

**Pull request closed without merging**